File: Msys_Programs/MSys_Projects/LeetCode/GoLang/RemoveDuplicatesfromSortedArray.go
package main

import "fmt"

func removeDuplicates(nums []int) {
	i := 1
	for i < len(nums) {
		if nums[i] == nums[i-1] {
			nums = append(nums[:i], nums[i+1:]...)
		} else {
			i++
		}
	}
	fmt.Println(i)
}

func main() {
	O := []int{2, 2, 3, 3, 4, 4, 5, 5, 6}
	removeDuplicates(O)
}

File: Msys_Programs/MSys_Projects/LeetCode/Python/RemoveDuplicatesfromSortedArray.py
class Solution:
    def removeDuplicates(self, nums):
        nums.extend(sorted(set(nums) ,reverse= True))
        nums.reverse()
        print(list(set(nums)))
    

O = Solution()
# L = [0,0,1,1,1,2,2,3,3,4]
L = [2,2,3,3,4,4,5,5,6]
print("Orignal list: ",L)
p = O.removeDuplicates(L)
print(p)